pystiche_papers/ulyanov_et_al_2016/_loss.py

```diff
@@ -45,6 +51,10 @@ def content_loss(
 ) -> FeatureReconstructionOperator:
     if score_weight is None:
         if impl_params:
+            # instance_norm:
+            # https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/train.lua#L54
+            # not instance_norm:
+            # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/stylization_train.lua#L22
             score_weight = 1e0 if instance_norm else 6e-1
         else:
             score_weight = 1e0
```

Comment:
No comment is needed if the value matches the paper

```suggestion
        # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/stylization_train.lua#L22
        score_weight = 6e-1 if impl_params and not instance_norm else 1e0
```